### ISSUE TITLE
Neovim's 'exrc' option is not respected

### ISSUE THREAD

**Issue by lottspot** [LunarVim/LunarVim#4242]
### Problem description

1. Enable 'exrc' option in `~/.config/lvim/config.lua`

    ```lua
    vim.o.exrc = true
    ```
2. Create a `.nvim.lua` in the current directory

    ```lua
    print("loaded .nvim.lua")
    ```

3. Open a file

    ```bash
    lvim noop.txt
    ```

4. Observe that there is nothing in `:messages`

When these steps are repeated by writing to `~/.config/nvim/init.lua` in step 1 and invoking vanilla `nvim` in step 3, the expected trust prompt is presented when the editor opens and the expected output can be observed in `:messages`.

### LunarVim version

release-1.3/neovim-0.9-0aea5a68

### Neovim version (>= 0.9.0)

NVIM v0.9.1

### Terminal name

tmux-256color

### Operating system/version

Arch Linux (last upgraded 10 June)

### Steps to reproduce

See above

**Comment by breneser** [comment]
Can confirm the same. It doesn't seem to be working.

**Comment by PizzasBear** [comment]
Neovim requires the `exrc` option to be set to use `.nvim.lua`, check `:h exrc`.
```lua
vim.opt.exrc = true
```

**Comment by garrettn** [comment]
> Neovim requires the `exrc` option to be set to use `.nvim.lua`, check `:h exrc`.
> 
> ```lua
> vim.opt.exrc = true
> ```

Normally that should work (and it does for me in vanilla Neovim), but setting this option in the LunarVim config file (`~/.config/lvim/config.lua`) appears to have no effect.

I got it working by using the [exrc.nvim](https://github.com/MunifTanjim/exrc.nvim) plugin, but that has been archived because of Neovim's built-in support. It would be nice to be able to get this functionality without a plugin. I'm not sure what in LunarVim's setup is keeping it from working.

**Comment by lottspot** [comment]
@PizzasBear I think I made it pretty clear in the description that setting that option does not work with LunarVim, and is in fact the exact reason this issue has been opened.

**Comment by Fsmash** [comment]
@lottspot 

At the core of this issue is that all of the LunarVim configurations are passed with "-u" to Neovim. Which skips all initialization past step 5 (see :h initialization). Even doing 

`$ lvim --cmd "set exrc"`

won't work since "-u" still skips initialization which is where exrc initialization built into Neovim also happens. 

I suspect but have not tested it out myself that a potential "fix" that would be a total hack is by setting "$VIMIINIT" by doing

`$ nvim --cmd "lua require("lvim")`

and making the appropriate additions to package.path in the Neovim config. 

Essentially , what I gather from the doc (:h MYVIMRC) is that getting LunarVim to work with the native implementation of exrc is fubar lol. Unless there is a way to set "$MYVIMRC" that circumvents Neovim setting it or invoking the exrc initialization directly somehow.

tl;dr If you don't want to add in another plugin to your setup, a quick an easy workaround is setting up an autocommand yourself to source whatever files you specify in the current directory. Just make sure you vet anything you add to your exrc config.

**Comment by JWambaugh** [comment]
I am having this issue too. It would be very nice if this could be fixed.
